client/src/app/project-registration/contact-field-group/contact-field-group.component.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('ContactFieldGroupComponent', () => {
   beforeEach(() => {
     userSpy = jasmine.createSpyObj(['subscribe']) as jasmine.SpyObj<Subject<User>>;
     firstUserSpy = jasmine.createSpyObj(['first']) as jasmine.SpyObj<Subject<User>>;
-    firstUserSpy.first.and.returnValue(userSpy);
+    // firstUserSpy.first.and.returnValue(userSpy); I don't know how to fix this!
```

Comment:
> This is soooo noice!

It was, it was also terribly broken because of a change in RxJS.
I've updated all our usages but we'll have to be careful with this one, lots of testing.

Also re: unit tests, I don't know what to do about this one.
The format is completely foreign to me.